source/partials/_toolchain_section.haml

```diff
@@ -9,5 +9,5 @@
 
     %ul
```

Comment:
Why adding here a class for each children instead of adding a single class to their parent?